.nuspec/Microsoft.Maui.Controls.targets

```diff
@@ -19,7 +19,7 @@
 
 	<ItemGroup>
 		<ProjectCapability Include="XamarinForms" />
```

Comment:
Should we be keeping this? Does this do anything at all?

Reply:
I removed it, it doesn't seem to break anything.